### PR TITLE
add Neel Trivedi member

### PR DESCRIPTION
**Pull request by makoteq**
fix for https://github.com/fossasia/fossasia.org/issues/562

### CODE REVIEW

**Comment by kushthedude** [comment]
Can you please also publish the site to gh-pages?

**Comment by makoteq** [comment]
> Can you please also publish the site to gh-pages?

I changed the name of repository to makoteq.github.io but it's dont work, what i do wrong
https://github.com/makoteq/makoteq.github.io

https://makoteq.github.io/

**Comment by kushthedude** [comment]
> > Can you please also publish the site to gh-pages?
> > I changed the name of repository to makoteq.github.io but it's dont work, what i do wrong
> > https://github.com/makoteq/makoteq.github.io
> 
> https://makoteq.github.io/

You dont need change repo name, just go to the fork settings and click on publish to gh-pages 🙂

**Comment by makoteq** [comment]
OK now it works - https://makoteq.github.io/fossasia.org/.
Thanks a lot for help

**Comment by makoteq** [comment]
So, can I submit task now?

**Review by nikit19 — APPROVED** [review]

**Comment by kushthedude** [comment]
@liveHarshit Review and merge the following.

**Review by kushthedude — APPROVED** [review]

**Review by liveHarshit — CHANGES_REQUESTED** [review]
Do not add members in the middle. Follow the sequence.

**Comment by liveHarshit** [comment]
And why you remove this file: `img/team/VinaySajjanapu.jpg` ?

File: team/index.html [hunk 1]
```diff
@@ -588,7 +588,20 @@ <h1>Developers</h1>
...
 					</div>
 				</div>
-
+				<div class="col-md-3 col-sm-6 speaker-column">
+					<div class="speaker">
+						<div class="image-holder">
+							<img class="background-image" alt="Neel Trivedi" src="../img/team/NeelTrivedi.jpg">
+							<div class="hover-state text-center preserve3d">
+								<div class="social-links vertical-align">
+									<a href="https://github.com/neel1998"><i class="icon fa fa-github"></i></a>
```
Comment (liveHarshit) [inline team/index.html]:
```suggestion
									<a href="https://github.com/neel1998" target="blank"><i class="icon fa fa-github"></i></a>
```

**Review by liveHarshit — CHANGES_REQUESTED** [review]

File: team/index.html (hunk 1 above)
Comment (makoteq) [reply team/index.html]: OK i changed sentence and added img/team/VinaySajjanapu.jpg photo

**Comment by makoteq** [comment]
Now its ok?
@liveHarshit

File: team/index.html [hunk 2]
```diff
@@ -2209,22 +2222,6 @@ <h1>Alumni</h1>
...
-										<a href="https://github.com/vinaysajjanapu"><i class="icon fa fa-github"></i></a>
-										<a href="https://twitter.com/sajjanapu_vinay" target="_self"><i class="icon social_twitter"></i></a>
-										<a href="https://www.linkedin.com/in/vinaysajjanapu"><i class="icon social_linkedin"></i></a>
-									</div>
-								</div>
-							</div>
-							<span class="speaker-name">Vinay Sajjanapu</span>
-							<span>Android Developer</span>
-						</div>
-					</div>
```
Comment (liveHarshit) [inline team/index.html]: Why these changes?

File: team/index.html [hunk 3]
```diff
@@ -588,7 +588,6 @@ <h1>Developers</h1>
 						<span>PDX Git Engineer</span>
 					</div>
 				</div>
-
```
Comment (liveHarshit) [inline team/index.html]: Why?

**Review by liveHarshit — CHANGES_REQUESTED** [review]

File: team/index.html (hunk 3 above)
Comment (makoteq) [reply team/index.html]: because first I wanted to add another team member but i changed my choice

Comment (liveHarshit) [reply team/index.html]: Revert unrelated changes

File: team/index.html (hunk 2 above)
Comment (makoteq) [reply team/index.html]: I deleted MY old code

Comment (liveHarshit) [reply team/index.html]: Here you removed one team member

**Comment by makoteq** [comment]
I Revert unrelated changes, now it's should be ok 
@liveHarshit

**Review by liveHarshit — APPROVED** [review]
Hope you got something to learn. :)

**Review by makoteq — COMMENTED** [review]
> Hope you got something to learn. :)

Of course!, and thanks for patient
